docs/exercises/flow.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Making an exercise
+sidebar_position: 2
+---
+
+## Types of exercises
+
+We have 3 different types of exercises:
+
+- [Theory](introduction)
+- [Practice](introduction)
+- [Self Diagnostic](introduction#self-diagnostic)
+
+This page only applies to the practice exercises, as this does not apply for the theory exercises and self diagnostics.
```

Comment:
Waarom niet ff linken naar self diagnostic in `index.md`

Reply:
Die bestaat niet toch?

`introduction#self-diagnostic` is het toch?

Reply:
`introduction#self-diagnostic` zou moeten werken